scripts/python/lib/validate_config_logic.py

```diff
@@ -87,6 +87,123 @@ def _validate_netmask_prefix(self):
     def validate_config_logic(self):
         """Config logic validation"""
 
+        from lib.config import Config
+
         self._validate_version()
         self._validate_netmask_prefix()
-        self.log.info('Config logic validation completed successfully')
+
+        CFG = Config(self.config)
+
+        def _validate_physical_interfaces():
```

Comment:
This private method should be moved out of the public method and called in the same manner as the two other private methods.

Reply:
Done